ml-package-versions.yml

```diff
@@ -0,0 +1,151 @@
+sklearn:
+  package_info:
+    pip_release: "scikit-learn"
+    install_dev: |
+      pip install git+https://github.com/scikit-learn/scikit-learn.git@master
+
+  models:
+    minimum: "0.20.4"
+    maximum: "0.23.2"
+    run: |
+      pytest tests/sklearn/test_sklearn_model_export.py --large
+
+  autologging:
+    minimum: "0.20.4"
```

Comment:
Looks like we use `0.20.3` as the minimum version in our current codebase: https://github.com/mlflow/mlflow/blob/2c77d25a2b699053e7b22253c17b63dfd5b5b6da/mlflow/sklearn/utils.py#L27. Do we need 0.20.4 instead?